src/contacts/view/ContactViewer.js

```diff
@@ -257,11 +257,15 @@ export class ContactViewer {
 	}
 
 	_formatBirthday(): string {
+		let bday = ""
 		if (this._hasBirthday()) {
-			return formatBirthdayWithMonthName(isoDateToBirthday(neverNull(this.contact.birthdayIso)))
-		} else {
-			return ""
+			try {
+				bday = formatBirthdayWithMonthName(isoDateToBirthday(neverNull(this.contact.birthdayIso)))
```

Comment:
This is invoked on every render which is already bad enough but if we try/catch/console.log it's going to heat entire district.
Maybe if we are here we should put it into some `memoized` thingy.

Reply:
yes, that's true. thanks for the pointer to the `memoized` stuff